.ci/deps.sh

```diff
@@ -15,7 +15,7 @@ npm install
 # GO commands
 go get -u github.com/golang/lint/golint
 go get -u golang.org/x/tools/cmd/goimports
-go get -u sourcegraph.com/sqs/goreturns
+go get -u github.com/sqs/goreturns
```

Comment:
has this to do with your issue?

Reply:
It has nothing to do with the issue.But the travis build was failing due to the removed link which I replaced it according to this link.https://github.com/sqs/goreturns/issues/23

Reply:
Then new issue or new commit explaining why you did this change